wgsl/index.bs

```diff
@@ -8494,19 +8494,11 @@ See [[#function-calls]].
   <thead>
     <tr><th>Parameterization<th>Overload<th>Description
   </thead>
-  <tr algorithm="isNan">
-  <td rowspan=4>
+  <tr algorithm="isFinite">
```

Comment:
Shouldn't we remove isFinite too?

Reply:
Discussed in #2309. 

We have agreement on isNan and isFinite. I'd rather land this and then do a second MR for isFinite.